does not build

go get github.com/bcicen/tcolors
# github.com/bcicen/tcolors/state
go/src/github.com/bcicen/tcolors/state/config.go:107:22: cannot use noire.NewRGB(float64(pc.RGB[0]), float64(pc.RGB[1]), float64(pc.RGB[2])) (type noire.Color) as type *noire.Color in return argument
go/src/github.com/bcicen/tcolors/state/config.go:112:22: cannot use noire.NewHSV(pc.HSV[0], pc.HSV[1], pc.HSV[2]) (type noire.Color) as type *noire.Color in return argument
go/src/github.com/bcicen/tcolors/state/config.go:114:22: cannot use noire.NewHex(pc.HEX) (type noire.Color) as type *noire.Color in return argument
go/src/github.com/bcicen/tcolors/state/state.go:48:39: cannot use noire.NewHSV(0, 0, 0) (type noire.Color) as type *noire.Color in field value
go/src/github.com/bcicen/tcolors/state/state.go:53:40: cannot use noire.NewHSV(hue, 80, 100) (type noire.Color) as type *noire.Color in field value
go/src/github.com/bcicen/tcolors/state/state.go:156:63: cannot use noire.NewHSV(s.Hue(), 100, 100) (type noire.Color) as type *noire.Color in return argument
go/src/github.com/bcicen/tcolors/state/substate.go:16:31: cannot use noire.NewRGB(128, 128, 128) (type noire.Color) as type *noire.Color in field value
go/src/github.com/bcicen/tcolors/state/substate.go:66:11: cannot use noire.NewHSV(n, s, v) (type noire.Color) as type *noire.Color in assignment
go/src/github.com/bcicen/tcolors/state/substate.go:72:11: cannot use noire.NewHSV(ss.hue, n, v) (type noire.Color) as type *noire.Color in assignment
go/src/github.com/bcicen/tcolors/state/substate.go:78:11: cannot use noire.NewHSV(ss.hue, s, n) (type noire.Color) as type *noire.Color in assignment
go/src/github.com/bcicen/tcolors/state/substate.go:78:11: too many errors


using arch
 5.10.6-arch1-1 #1 SMP PREEMPT Sat, 09 Jan 2021 18:22:35 +0000 x86_64 GNU/Linux
go version
go version go1.15.6 linux/amd64

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/cckuailong/colorsys-go v0.0.0-20190129090037-221d67bbdb7c
+	github.com/cckuailong/colorsys-go v0.0.0-20200114124858-f94465f7d6be // indirect
 	github.com/fatih/color v1.7.0
 	github.com/gdamore/tcell v1.1.2
 	github.com/mattn/go-colorable v0.1.2 // indirect

